### PR TITLE
Create filter for desktop

### PR DESCRIPTION
**Pull request by GiovannaValadao**
Create filter for desktop:

Captura de Tela 2022-02-18 às 01 23 05

### CODE REVIEW

File: src/components/filter/filter.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  margin: 10px;
+  align-items: center;
+  justify-content: center;
+}
+.filter-container__list {
+  display: flex;
+  gap: 20px;
+  list-style: none;
+}
+.filter-container__list-item {
```
Comment (raianechristine) [inline src/components/filter/filter.css:13]: what do you think about adding a hover effect? like a new background-color

File: src/components/filter/Filter.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+import "./filter.css";
+
+function Filter() {
+  return (
+    <div class="filter-container">
+      <ul class="filter-container__list">
+        <li class="filter-container__list-item">
+          <span class="filter-container__list-text">Ordenar</span>
+          <span class="image">
```
Comment (raianechristine) [inline src/components/filter/Filter.jsx]:
```suggestion
          <span class="filter-container__image">
```

File: src/components/filter/filter.css (hunk 1 above)
Comment (GiovannaValadao) [reply src/components/filter/filter.css:13]: Good! Added hover effect:

https://user-images.githubusercontent.com/42494406/155044817-71f29efd-d34c-4f71-9f7c-235daeafe4bd.mov

File: src/components/filter/Filter.jsx (hunk 2 above)
Comment (GiovannaValadao) [reply src/components/filter/Filter.jsx]: I changed, tks <3

File: src/components/filter/filter.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  cursor: pointer;
+}
+.filter-container__list-item:hover {
+  background-color: #f7f7f7;
+}
+.filter-container__list-text {
+  display: flex;
+  align-items: center;
+}
+.filter-container__image {
```
Comment (raianechristine) [inline src/components/filter/filter.css:29]: you changed this class but changed in only one div, then, the layout now is not correct 🤔  

image

image

Comment (GiovannaValadao) [reply src/components/filter/filter.css:29]: Really 😞 

I changed the other classes:

Captura de Tela 2022-02-22 às 21 20 09

**Review by raianechristine — APPROVED** [review]

**Pull request merged**